### ISSUE TITLE
adding erfinv in elementwise 

### ISSUE THREAD

**Issue by MuhammadNizamani** [unifyai/ivy#16720]
I am going to erfinv in elementwise.py from  #761 I know this todo list is done but this function must be in elementwise so I can erfinv in paddle.tesnor.tensor.Tensor .

**Comment by MuhammadNizamani** [comment]
@sagewhocodes thanks for that infromation

**Comment by AnnaTz** [comment]
Hi @MuhammadNizamani, I have added `ivy.erfinv` to our list here https://github.com/unifyai/ivy/issues/3856
Feel free to assign yourself to that task!

**Comment by MuhammadNizamani** [comment]
@AnnaTz so I should create new issue for erginv and then track that in this https://github.com/unifyai/ivy/issues/3856 ?

**Comment by AnnaTz** [comment]
> @AnnaTz so I should create new issue for erginv and then track that in this #3856 ?

Yes you should open an issue linked to that list and then make a PR for that.

**Comment by MuhammadNizamani** [comment]
@AnnaTz one more things what should be the label of new issue that I wll about erfinv

**Comment by AnnaTz** [comment]
> @AnnaTz one more things what should be the label of new issue that I wll about erfinv

`Sub Task`
